specification/security/resource-manager/Microsoft.Security/preview/2019-01-01-preview/serverVulnerabilityAssessments.json

```diff
@@ -0,0 +1,335 @@
+                        "required": true,
+                        "type": "string"
+					},
+                    {
+                        "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+                    }
+                ],
+                "responses": {
+					"200": {
+                        "description": "OK",
+                        "schema": {
+                            "$ref": "#/definitions/ServerVulnerabilityAssessment"
+                        }
+                    },
+                    "default": {
+                        "description": "Error response describing why the operation failed.",
+                        "schema": {
+                            "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+                        }
+                    }
+                }
+			},
+			"put": {
+				"x-ms-examples": {
+                    "Create a server vulnerability assessments on a resource. Only 'default' resource is supported. Once creating the resource, the server will be onboarded to vulnerability assessment by Microsoft.Security": {
+                        "$ref": "./examples/ServerVulnerabilityAssessments/CreateServerVulnerabilityAssessments_example.json"
+                    }
+                },
+				"description": "Creating a server evulnerability assessment on a resource, which will onboard a resource for having a vulnerability assessment on it",
+                "tags": [
+                    "ServerVulnerabilityAssessment"
+                ],
+                "operationId": "ServerVulnerabilityAssessment_CreateOrUpdate",
+                "parameters": [
+                    {
+                        "$ref": "../../../common/v1/types.json#/parameters/SubscriptionId"
+                    },
+                    {
+                        "$ref": "../../../common/v1/types.json#/parameters/ResourceGroupName"
+                    },
+                    {
+                        "name": "resourceNamespace",
+                        "in": "path",
+                        "description": "The Namespace of the resource.",
+                        "required": true,
+                        "type": "string"
+                    },
+                    {
+                        "name": "resourceType",
+                        "in": "path",
+                        "description": "The type of the resource.",
+                        "required": true,
+                        "type": "string"
+                    },
+                    {
+                        "name": "resourceName",
+                        "in": "path",
+                        "description": "Name of the resource.",
+                        "required": true,
+                        "type": "string"
+                    },
+					{
+						"name": "serverVulnerabilityAssessment",
+                        "in": "path",
```

Comment:
You should add an enum with `default` as the only value. See enum here: https://swagger.io/docs/specification/2-0/describing-parameters/